### PR TITLE
final task_google clone

### PR DESCRIPTION
**Pull request by arinzeatusiuba**
This is my final task for lessons 1 and 2, please review @ChidiPatrick @promise-J @OgJaykay
@Pieta9221 @MiniHack @Melumm
please kindly note that our instructor asked that I avoid the lint error check

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <title>Basic page of google</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a class="small-text" href="#">Gmail</a></li>
+                <li><a class="small-text" href="#">Images</a></li>
+                <li class="menu-bg">
+                    <img class="menu-button" src="/image/apps_FILL0_wght400_GRAD0_opsz48.png" alt="menu-button">
```
Comment (Pieta9221) [inline html/index.html]: Fix this path

File: html/index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
```
Comment (Pieta9221) [inline html/index.html]: Fix this path

File: css/style.css [hunk 3]
```diff
@@ -1,52 +1,142 @@
...
+    height: 20px;
+    margin-top: 6px;
+ border-radius: 50%; 
+ background-color: royalblue
 }
 
-table {
-	border-collapse: collapse;
-	border-spacing: 0;
+.search-wrapper {
```
Comment (Pieta9221) [inline css/style.css:58]: It should be centralized as it looks more to the left

File: html/index.html (hunk 1 above)
Comment (Pieta9221) [inline html/index.html]: Fix the file path. Do the same for all images

File: html/index.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        <div class="panel">
+            <ul>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Advertising</a></li>
+                <li><a href="#">Business</a></li>
+                <li><a href="#">How search work</a></li>
+            </ul>
+            <div class="carbon"> 
+                <img class="leaf" src="/image/Leaves.png" alt="leaf">
+                <a href="#">Carbon neutral since 2007</a>
```
Comment (Pieta9221) [inline html/index.html:59]: Centralize this

**Review by Pieta9221 — COMMENTED** [review]
😐😐😐
Your work looks good if the images and stylesheet are properly linked
Kindly review HTML absolute and relative paths. 
It will help in fixing the major problem.
Additionally, centralize the search bar and the writings beneath.
Let us know if you encounter any challenges
There is always room for improvement!
After making the corrections, merge into main branch and don't forget to host on GitHub pages

File: html/index.html [hunk 5]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
+    <title>Basic page of google</title>
```
Comment (Pieta9221) [inline html/index.html]: Rename to Google Clone

Comment (arinzeatusiuba) [reply html/index.html]: I have made all the changes on google clone

**Comment by Pieta9221** [comment]
Good Job!Go ahead and merge?

Sent from Yahoo Mail on Android 
 
  On Wed, May 10, 2023 at 1:52 PM, ***@***.***> wrote:   


@arinzeatusiuba commented on this pull request.

In html/index.html:
> @@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
+    <title>Basic page of google</title>

I have made all the changes on google clone

File: html/index.html [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    <title>Google Clone</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a class="small-text" href="#">Gmail</a></li>
+                <li><a class="small-text" href="#">Images</a></li>
+                <li class="menu-bg">
+                    <img class="menu-button" src="../image/apps_FILL0_wght400_GRAD0_opsz48.png" alt="menu-button">
```
Comment (ChidiPatrick) [inline html/index.html:17]: close your img tag properly

File: html/index.html [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+                </li>
+                <li>
+                    <img class="profile-pic" src="../image/20211027_221854.png" alt="profile-pic">
```
Comment (ChidiPatrick) [inline html/index.html:20]: close the element properly

**Review by ChidiPatrick — COMMENTED** [review]
Close your elements properly

**Comment by arinzeatusiuba** [comment]
https://github.com/arinzeatusiuba/Lesson_1_-_2_Final_task_Arinze-Atusiuba/pull/2

I have closed the img tag properly
Thank You

**Pull request merged**